search/grammar_v2_engine.go

```diff
@@ -302,3 +305,97 @@ func (e *ESEngine) searchResultsToIntents(query *Query, language string, result
 	// log.Infof("Intents: %+v", intents)
 	return normalizedIntents, nil
 }
+
+func (e *ESEngine) ConventionsLandingPageToCollectionHit(year string, location string) (*elastic.SearchHit, error) {
```

Comment:
What are those three function do?
Why are they needed?
How come I did not need this for conventions yearlier?

Reply:
in case we have a LP with only one collection inside, we want to convert it to that collection in the results...